simpleclient_guava/src/main/java/io/prometheus/client/cache/guava/CacheMetricsCollector.java

```diff
@@ -0,0 +1,79 @@
+package io.prometheus.client.cache.guava;
```

Comment:
@checketts For Caffeine, you can either pull metrics from CacheStats (like Guava) or push into metrics using a custom `StatsCollector`. See [stats-metrics](https://github.com/ben-manes/caffeine/tree/master/examples/stats-metrics) for a Dropwizard example of the push approach. Both are equivalent and the choice depends on preferences.